### PR TITLE
Rules cleanup and terminology updates

### LINKED ISSUE #11989: Conflicting information about tournament bans
The page [Tournament bans](/ppy/osu-wiki/tree/master/wiki/Help_centre/Tournament_bans/en.md) states that tournament bans are appealable, but the page [Account restrictions](/ppy/osu-wiki/tree/master/wiki/Help_centre/Account_restrictions/en.md) states that tournament bans cannot be appealed. This discrepancy should be cleared up, it should specify when a tournament ban may be reconsidered for appeal and when it cannot on both pages.

### PR DESCRIPTION
**Pull request by Userlog2**
(clayton edit) closes #11989

## Self-check

- [X] The changes are tested against the [contribution checklist](https://osu.ppy.sh/wiki/osu!_wiki/Contribution_guide#self-check)

### CODE REVIEW

File: wiki/Reporting_bad_behaviour/Handling_foul_play/en.md [hunk 1]
```diff
@@ -41,13 +41,13 @@ You should report the user and not mention them anywhere. There are three ways t
...
 
-On the user's profile, a `Report` button will appear when you click on the three dots next to the `Message` button. Write any information that can prove that the user is doing misbehaviour.
+On the user's profile, a `Report` button will appear when you click on the three dots next to the `Message` button. Write any information that can prove that the user is misbehaving.
 
 *Take note of the selected game mode before reporting.*
 
 ### Beatmap page
 
-This way of reporting is also as effective as using the profile page for voting, the reports go to the same endpoint.
+This way of reporting is as effective as using the profile page for reporting - the reports go to the same endpoint.
```
Comment (Walavouchey) [inline wiki/Reporting_bad_behaviour/Handling_foul_play/en.md:50]: em dash here

```suggestion
This way of reporting is as effective as using the profile page for reporting – the reports go to the same endpoint.
```

File: wiki/Help_centre/Account_restrictions/en.md [hunk 2]
```diff
@@ -71,11 +71,11 @@ If we decide to let you return, we will require that you agree to some terms bef
...
+Appeal returnees under the standard terms above are also [banned from official and badged tournaments](/wiki/Help_centre/Tournament_bans), both as a competitor and member of the tournament's team. The duration of this ban is 1 year per offence (if a user is on their second appeal, the ban lasts 2 years). The unrestriction email will clarify the duration of this ban.
 
-**Cheating or account sharing during an official or badged tournament will lead to the user's permanent ban from any future ones.** This ban cannot be appealed (your email will be ignored if you attempt to) and will not be removed, reduced, or waived under any circumstances.
+**Cheating or account sharing during an official or badged tournament will lead to the user's indefinite ban from any future ones.** This ban can only be appealed after two years, and will only be granted if the user has put significant effort into improving community tournaments.
```
Comment (cl8n) [inline wiki/Help_centre/Account_restrictions/en.md:76]:
```suggestion
**Cheating or account sharing during an official or badged tournament will lead to the user's indefinite ban from any future ones.** This ban can be appealed after two years, but will be granted only if the user has put significant effort into improving community tournaments.
```

**Review by cl8n — COMMENTED** [review]
seems good to me overall

File: wiki/Help_centre/Account_restrictions/en.md (hunk 2 above)
Comment (Userlog2) [reply wiki/Help_centre/Account_restrictions/en.md]: Forgot to change the suggestion apply message, whoops!

**Review by cl8n — APPROVED** [review]

**Review by peppy — APPROVED** [review]

**Pull request merged**